locales/en/moves.json

```diff
@@ -80,5 +80,14 @@
       "confirmation": "Are you sure you want to cancel the move for <strong>{{name}}</strong> to <strong>{{location}}</strong>?",
       "warning": "This will delete this move request."
     }
+  },
+  "confirmation": {
+    "page_title": "Move confirmation for {{name}}",
+    "panel": {
+      "heading": "Move scheduled",
+      "content": "Move reference<br><strong>{{moveReference}}</strong>"
```

Comment:
@lmoney-moj Could this just be `Reference number` now as we've used the term "Move" already in this panel?